src/jit/emitxarch.cpp

```diff
@@ -386,6 +386,9 @@ bool TakesRexWPrefix(instruction ins, emitAttr attr)
             case INS_movnti:
             case INS_pdep:
             case INS_pext:
+            case INS_bextr:
```

Comment:
nit: It looks like this list was ordered alphabetically.

Reply:
(same goes for some of the other lists below)

Reply:
Fixed, thanks!